pkg/providers/vsphere/tags.go

```diff
@@ -20,8 +20,12 @@ func requiredTemplateTags(clusterSpec *cluster.Spec, machineConfig *v1alpha1.VSp
 
 func requiredTemplateTagsByCategory(clusterSpec *cluster.Spec, machineConfig *v1alpha1.VSphereMachineConfig) map[string][]string {
 	osFamily := machineConfig.Spec.OSFamily
+	vb, err := clusterSpec.GetCPVersionsBundle()
```

Comment:
same here
this depends on the kubernetes version assigned to the machine config (cp vs workers)